packages/webapp/pages/onboarding.tsx

```diff
@@ -117,15 +196,23 @@ export function OnboardPage(): ReactElement {
     router.push('/');
   };
 
+  const onSuccessfulLogin = () => {
+    setFinishedOnboarding(true);
+    router.push('/');
```

Comment:
should this be a replace? do we want the user to be able to navigate back to the onboarding screen after logging in?
```suggestion
    router.replace('/');
```

Reply:
I've changed this on those created by V3, but should I also do it on the ones for V2 @rebelchris?

Reply:
You should not be able to navigate back technically.
For v2 it will hard force you back to `/` anyway. *(it's existing logic)

So we can simply add this in v2 as well yes.

Reply:
Done.